raster_loader/io/datawarehouse.py

```diff
@@ -0,0 +1,188 @@
+import pandas as pd
+
+from typing import List
+
+
+class DataWarehouse:
```

Comment:
I would rename it as `DataWarehouseConnection`

Reply:
Updated